.github/renovate.json5

```diff
@@ -13,7 +13,7 @@
   // The branches renovate should target
   // PLEASE UPDATE THIS WHEN RELEASING.
   "baseBranches": [
-    "master",
+    "main",
```

Comment:
ah, this would hopefully help the new warning that showed up on the renovate [dependency dashboard](https://github.com/crossplane/crossplane/issues/5574): `WARN: Base branch does not exist - skipping`

Renovate also closed a bunch of PRs on `master` at the same time it showed that warning (e.g., https://github.com/crossplane/crossplane/pull/5843 and https://github.com/crossplane/crossplane/pull/5780), I have to imagine they are related. Perhaps it will re-open them on the `main` branch after we merge these updates?

Reply:
@jbw976 its likely related yes.